rpmlint/descriptions/PythonCheck.toml

```diff
@@ -0,0 +1,25 @@
+python-tests-in-package="""
+test/ or tests/ directory in package directory. Tests should not be included in %{python_sitelib}/<pkgname>."""
+python-doc-in-package="""
+doc/ or docs/ directory in package directory. Documentation should go into %{docdir}, not %{python_sitelib}/<pkgname>"""
+python-src-in-package="""
+src/ directory included in %{python_sitelib}/<pkgname>. This should be in a separate -src package.
+"""
+python-egginfo-require-not-in-spec="""
+Requires pulled in by the spec do not match upstream requires.txt. Please double-check."""
```

Comment:
This needs to be removed.

Reply:
Why?

Reply:
It does, indeed - missed that one when removing the Requires check. Thanks for spotting!

Reply:
Removed.